sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ShowTablesExec.scala

```diff
@@ -33,7 +33,8 @@ case class ShowTablesExec(
     output: Seq[Attribute],
     catalog: TableCatalog,
     namespace: Seq[String],
-    pattern: Option[String]) extends V2CommandExec with LeafExecNode {
+    pattern: Option[String],
+    listTempViews: Option[String] => Seq[Identifier]) extends V2CommandExec with LeafExecNode {
```

Comment:
do we need to pass this? All `SparkPlan` can access the current spark session by calling `this.session`

Reply:
This is removed now.